entry/src/main/java/me/xdj/multistateview/ActivityUtils.java

```diff
@@ -0,0 +1,44 @@
+/*
```

Comment:
Remove the applibgroup copyright for this file. It is the same as Base library's ActivityUtils.java